Client/Application/AppDelegate.swift

```diff
@@ -294,6 +294,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             self.profile?.shutdown()
             application.endBackgroundTask(taskId)
         }
+
+        // Workaround for crashing in the background when <select> popovers are visible.
+        // See http://openradar.appspot.com/radar?id=5033179731722240
+        let jsBlurSelect = "if (document.activeElement.tagName === 'SELECT') { document.activeElement.blur(); }"
```

Comment:
WebKit used `<body>` in all cases I tried, but I'll add a check anyway to be safe.